packages/ui/__stories__/Badge.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import { Badge } from '../src/Badge'
+
+export default {
+  title: 'components/Badge',
+  component: Badge,
+}
+
+export const NeutralNormal = () => <Badge size="normal" type="neutral" content="Badge text" />
```

Comment:
nit: It seems like we support multi-line content. Should we add at least one story with the multi-line content?